Source/Classes/Editor/DZNPhotoEditorViewController.m

```diff
@@ -691,31 +691,27 @@ - (void)acceptEdition:(id)sender
         return;
     }
     
-    dispatch_queue_t exampleQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0);
-    dispatch_async(exampleQueue, ^{
+    dispatch_queue_t queue = dispatch_get_main_queue();
```

Comment:
Why do you need to use the main queue?

Reply:
This is the basic concept of the programming for iOS - the UI shouldn't be called from the Main thread. Otherwise, you would get the following run-time errors:

![Screen Shot 2020-01-22 at 10 17 15 AM](https://user-images.githubusercontent.com/53795708/72922040-2ef08e00-3d01-11ea-8d82-197e4cf66f96.png)